hera_cal/lst_stack/io.py

```diff
@@ -175,23 +221,17 @@ def create_lstbin_output_file(
     file_history = f"{history} Input files: {file_list_str}"
     _history = file_history + utils.history_string()
 
-    if lst < lst_branch_cut:
-        lst += 2 * np.pi
+    fname = fname.format(kind=kind)
 
-    fname = fname_format.format(
-        kind=kind,
-        lst=lst,
-        pol="".join(pols),
-        inpaint_mode=(
-            "inpaint" if inpaint_mode else ("flagged" if inpaint_mode is False else "")
-        ),
-    )
```

Comment:
I separated out the formatting into its own function, because I found I wanted to call it for other output files in the notebook (other than these uvdata files)